content/1.md

```diff
@@ -1,8 +1,103 @@
 ---
-title: 1 - Windows powertools & Firefox extensions
-description: Ensimmäinen episodi juhuu!
-date: 2021-01-01
+title: '1 – Windows powertools & Firefox extensions 🦾'
+description: |
+  Pyöräytetään uusi podi käyntiin metajaksolla!
+  Mitkä ovat devaajan parhaat tehotyökalut Windowsille?
+  Entä parhaat lisäosat Firefoxille?
+date: 2021-01-01 # TODO: Replace with real publication date
+recorded: 2021-03-19 # TODO: Is this correct?
```

Comment:
@Steellow Can you check this?

Reply:
Seems correct 👍

Reply:
On Discord you said that the date might be 22nd March. Can you double check please? 😃

Reply:
19th seems to be correct 😌